### DISCUSSION TITLE
set permission to join a server?

### DISCUSSION THREAD

**Discussion by nastaros** [Pryaxis/TShock#2183] [Q&A]
im running 3 servers sharing the same mysql database, is it possible to limit access to one of them to only users with a certain permission?

or any other option to limit access other than ip whitelist and password?

edit: I also tried making a region, and with smartregion executing /kick [playername], and giving a user group the nokick permission. but they still get kicked. is there a command for kicking specific groups like guest and default?

**Comment by bartico6** [comment]
There isn't an option to do this without custom plugins.

**Comment by moisterrific** [comment]
Would setting max slots to 0 and reserved slots to a specific number work? And of course giving the groups you want to allow the reserved permission.
